GrievanceManagement/src/main/java/com/gms/constants/VariableConstant.java

```diff
@@ -0,0 +1,28 @@
+package com.gms.constants;
+
+/**
+ * This is @ConstantVariable class for defining all the constant.
+ */
+public class VariableConstant {
+
+    /**
+     * This is minimum length of user name.
+     */
+    public static final int NAME_MIN_LENGTH = 2;
```

Comment:
Integer

Reply:
we can't change this value because the type of @NotEmpty min attribute is int